include/SDL3/SDL_gpu.h

```diff
@@ -121,8 +121,9 @@ typedef enum SDL_GpuTextureFormat
 	SDL_GPU_TEXTUREFORMAT_R16G16B16A16_UINT,
 	/* Depth Formats */
 	SDL_GPU_TEXTUREFORMAT_D16_UNORM,
```

Comment:
is d16 widely supported and it's just d16_s8 that isn't?

Reply:
Yeah, d16 is guaranteed to be supported by all APIs. For Vulkan it has 99.88% device coverage on gpuinfo, while d16_s8 has 22%.